web/Web/web/search/templates/search/search_list.html

```diff
@@ -82,31 +85,36 @@ <h6 class="title" style="margin-bottom: 0;">{{ hit.title|escape }}</h6>
       </div>
     </div>
   </div>
-  {% endif %}
 
 <nav class="my-5">
   <ul class="pagination justify-content-center">
       <li class="page-item {{ pagination.is_first_page|yesno:'disabled,' }}">
-      <a class="page-link" href="{% url 'search:query_result' query_id=queryID%}?page_number=1">First</a>
+      <a class="page-link" href="{% url 'search:main'%}?query={{query.split|join:'+'}}&page_number=1">First</a>
     </li>
     <li class="page-item {{ pagination.is_first_page|yesno:'disabled,' }}">
-      <a class="page-link" href="{% url 'search:query_result' query_id=queryID%}?page_number={{pagination.page_number|add:-1}}">Prev</a>
+      <a class="page-link" href="{% url 'search:main'%}?query={{query.split|join:'+'}}&page_number={{pagination.page_number|add:-1}}">Prev</a>
     </li>
     {% for i in pagination.page_range %}
     <li class="page-item {% if pagination.page_number == i %} active {% endif %}">
-      <a class="page-link" href="{% url 'search:query_result' query_id=queryID%}?page_number={{i}}">
+      <a class="page-link" href="{% url 'search:main'%}?query={{query.split|join:'+'}}&page_number={{i}}">
         {{i}}
         <span class="sr-only">(current)</span>
       </a>
     </li>
     {% endfor %}
     <li class="page-item {{ pagination.is_last_page|yesno:'disabled,' }}">
-        <a class="page-link" href="{% url 'search:query_result' query_id=queryID%}?page_number={{pagination.page_number|add:+1}}">Next</a>
-    </li>
-    <li class="page-item {{ pagination.is_last_page|yesno:'disabled,' }}">
-        <a class="page-link" href="{% url 'search:query_result' query_id=queryID%}?page_number={{pagination.last_page}}">Last</a>
+        <a class="page-link" href="{% url 'search:main'%}?query={{query.split|join:'+'}}&page_number={{pagination.page_number|add:+1}}">Next</a>
     </li>
 
   </ul>
 </nav>
+
+<style>
+  .document em {
```

Comment:
Let's move this to `static/css/project.css`. https://github.com/UWaterlooIR/gathera/blob/06d8b3e05907a6af6f47439288271f7b96531d9a/web/Web/web/static/css/project.css#L710